README.md

```diff
@@ -5,28 +5,32 @@
 [![GHA](https://img.shields.io/github/v/tag/iterative/setup-cml?label=GitHub%20Actions&logo=GitHub)](https://github.com/iterative/setup-cml)
 [![npm](https://img.shields.io/npm/v/@dvcorg/cml?logo=npm)](https://www.npmjs.com/package/@dvcorg/cml)
 
-**What is CML?** Continuous Machine Learning (CML) is an open-source library for
-implementing continuous integration & delivery (CI/CD) in machine learning
-projects. Use it to automate parts of your development workflow, including model
-training and evaluation, comparing ML experiments across your project history,
-and monitoring changing datasets.
+**What is CML?** Continuous Machine Learning (CML) is an open-source CLI tool
+for implementing continuous integration & delivery (CI/CD) with a focus on
+MLOps. Use it to automate parts of development workflows — including machine
+provisioning; model training and evaluation; comparing ML experiments across
+project history, and monitoring changing datasets.
 
-![](https://static.iterative.ai/img/cml/github_cloud_case_lessshadow.png) _On
-every pull request, CML helps you automatically train and evaluate models, then
-generates a visual report with results and metrics. Above, an example report for
-a [neural style transfer model](https://github.com/iterative/cml_cloud_case)._
+For example, on every pull request CML can help to automatically train and
+evaluate models, then generate a visual report with results and metrics.
```

Comment:
Just for the key paragraphs, I recommend being more direct, e.g.:
```suggestion
CML can help train and evaluate models automatically, and generate a visual
report with results and metrics on every pull request.
```

Reply:
Could even split the sentences for extra clarity.

Reply:
💯